src/images/buttons_180px.svg

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
FYI the update to the button front-page image and the relevant CSS is already in master now.